src/main/java/jimmy/mcgymmy/model/food/Fridge.java

```diff
@@ -53,6 +53,17 @@ public void remove(Index removeIndex) {
         internalList.remove(removeIndex.getZeroBased());
     }
 
+    /**
+     * Removes the food item
+     * The food item must exist in the list.
+     */
+    public void remove(Food toRemove) {
+        requireNonNull(toRemove);
+        if (internalList.contains(toRemove)) {
+            internalList.remove(toRemove);
+        }
```

Comment:
I think its good to add an assertion if the food to be removed does not exist as checking for existence should occur before actually removing the food.

Reply:
Remove this method then  should be good to merge